Fix get_ofx.js. Closes #16

jwheare opened the pull request.

russss commented:
I'm a bit stumped by this, as get_ofx works fine for me as-is.

This branch fails for me with the error:
`[error] [remote] mouseEvent(): Couldn't find any element matching '#home' selector`

jwheare commented:
Alright, should work for you now.

* Simplified account fetching
* Always fetch account indexes even for configured accounts, just use them for limiting/aliasing
* Made all exported files save to `export/`
* Changed the `onAccounts` api, it receives a single object of accounts now looking like this:
```
{
  '12345': {
    number: '12345',
    name: '12345 OR config key alias if set'
    idx: '0'
  }
  ...
}
```

russss commented:
Yup, that works fine now, thanks!
